examples/todo/js/subscriptions/AddTodoSubscription.js

```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```

Comment:
Ah, okay, Relay does it for mutations only in https://github.com/facebook/relay/pull/657.